telegram/constants.py

```diff
@@ -66,11 +84,30 @@
 ]
 
 from enum import IntEnum
-from typing import List
+from typing import List, NamedTuple
 
 from telegram._utils.enum import StringEnum
 
-BOT_API_VERSION = "6.0"
+
+class _BotAPIVersion(NamedTuple):
+    """Similar behavior to sys.version_info.
+    So far TG has only published X.Y releases. We can add X.Y.Z(a(S)) if needed.
+    """
+
```

Comment:
what happens if we add `__slots__` here?

Reply:
`AttributeError: Cannot overwrite NamedTuple attribute __slots__` 🙆‍♀️

Reply:
interesting